guides/hack/00-overview/03-async.md

```diff
@@ -1,85 +1,15 @@
 ### Async Programming in Hack
 
-Async (or asynchronous to give the full title) programming is, at the very basic level, a way to have pieces of code that involve a lot of waiting (such as making a GET request to a remote web server, or database queries) run without blocking other parts of the code from executing. 
+Async (synchronous) programming is a way to have pieces of code that involve waiting (such as making a GET request to a remote web server, or database queries) to run without blocking other parts of the code from executing. 
 
-#### A Basic Example
+Imagine you need the result of two functions, `foo()` and `bar()`; `foo()` requires data from a web page or HTTP API, and `bar()` needs data from MySQL.
 
-Here's a really simple piece of code that would fetch the content of a URL and then does something with the result:
+Traditionally, the execution flow will look like this:
 
-```
-$x = makeGetRequest('http://example.com/a/');
-$y = makeGetRequest('http://example.com/b/');
+![Synchronous execution](/images/async/feature-advert-sync.png) 
 
-echo $x;
+Async allows performance improvements by changing the flow to look like this instead:
 
-function makeGetRequest(string $url): string {
-  $ch = curl_init($url);
-  curl_setopt($ch, CURLOPT_HEADER, 0);
-  $content = curl_exec($ch);
-  curl_close($ch);
-  return $content;
-}
+![Asynchronous execution](/images/async/feature-advert-async.png)
 

```

Comment:
Further Reading will be good for consistency across the overviews as it's always there. I don't think it matters how short this page is. 

I would reword the leading sentence to say:

"Hack's Async functionality introduces the potential for performance improvements by changing the flow to look like this instead:"